src/main/java/io/kuzzle/sdk/core/Kuzzle.java

```diff
@@ -246,12 +249,13 @@ public Kuzzle(@NonNull final String url, final KuzzleOptions options, final Kuzz
     this.defaultIndex = opt.getDefaultIndex();
     this.headers = opt.getHeaders();
     this.metadata = opt.getMetadata();
+    this.port = opt.getPort();
     this.queueMaxSize = opt.getQueueMaxSize();
     this.queueTTL = opt.getQueueTTL();
     this.reconnectionDelay = opt.getReconnectionDelay();
     this.replayInterval = opt.getReplayInterval();
 
-    this.url = url;
+    this.url = "http://" + host + ":" + this.port;
```

Comment:
As for PHP SDK, the protocol should be parametrable, in order to enable to reach kuzzle behind a SSL proxy
(and I wonder if we should use "ws / wss" protocol prefix instead of "http / https", to be compliant with the RFC ?)